mplc/partner.py

```diff
@@ -41,3 +41,50 @@ def corrupt_labels(self):
     def shuffle_labels(self):
         for label in self.y_train:
             label = shuffle(label)
+
+
+class PartnerMpl:
+
+    def __init__(self, partner, mpl):
+        """
+        :type partner: Partner
+        :type mpl: MultiPartnerLearning
+        """
+        self.mpl = mpl
+        self.id = partner.id
+        self.batch_size = partner.batch_size
+        self.minibatch_count = mpl.minibatch_count
+        self.partner_parent = partner
```

Comment:
For coherence and readibility shouldn't we call the argument `parent` and the attribute `self.partner_parent` with the same name instead of 2 different ones?